PixivBrowserFactory.py

```diff
@@ -1019,7 +1020,7 @@ def sketch_get_posts_by_artist_id(self, artist_id, max_page=0):
 
         return artist
 
-    def getMangaSeries(self, manga_series_id: int, current_page: int) -> PixivMangaSeries:
+    def getMangaSeries(self, manga_series_id: int, current_page: int, returnJSON = False) -> PixivMangaSeries:
```

Comment:
can you change the return type from `PixivMangaSeries` to `Union[PixivMangaSeries, str], because this function now can return PixivMangaSeries object or json string (actually getPixivPage also can return BeautifulSoup object, but based on url, it should only return json string)